understanding/20/focus-order.html

```diff
@@ -134,7 +134,8 @@ <h2>Examples of Focus Order</h2>
          
          <li>A web page implements modal dialogs via scripting. When the trigger button is activated,
```

Comment:
```suggestion
         <li>A web page implements modal dialogs. When the trigger button is activated,
```
Now that we’ve got the declarative [`command`](https://html.spec.whatwg.org/multipage/form-elements.html#attr-button-command) / [`commandfor`](https://html.spec.whatwg.org/multipage/form-elements.html#attr-button-commandfor) approach, “_via scripting_” is no longer a given?